src/gui/foldercreationdialog.cpp

```diff
@@ -0,0 +1,80 @@
+#include "foldercreationdialog.h"
+#include "ui_foldercreationdialog.h"
+
+#include <limits>
+
+#include <QDir>
+#include <QMessageBox>
+
+namespace  {
+  const QString SuggestedFolderNamePrefix = QObject::tr("New folder");
```

Comment:
Better not have such a global static on a non-POD type. Also this is really only used from one function AFAICT so better have it as a local variable there (also needs to start in lowercase).

Reply:
Still needs to be moved as a local variable in the ctor.